yarn-project/archiver/src/archiver/kv_archiver_store/kv_archiver_store.ts

```diff
@@ -63,6 +67,14 @@ export class KVArchiverDataStore implements ArchiverDataStore {
     return (await Promise.all(data.map(c => this.#contractClassStore.addContractClass(c)))).every(Boolean);
   }
 
+  async addPrivateFunctions(
+    contractClassId: Fr,
+    privateFunctions: ExecutablePrivateFunctionWithMembershipProof[],
+  ): Promise<boolean> {
+    await this.#contractClassStore.addPrivateFunctions(contractClassId, privateFunctions);
+    return Promise.resolve(true);
```

Comment:
Should this return value mean something? Maybe the function we forward to should return something. Otherwise we can presumably just return void.